pep-0696.rst

```diff
@@ -206,20 +206,26 @@ default to the type of ``start`` and step default to ``int | None``.
    class Foo(Generic[DefaultStrT, T2]):
        def __init__(self, a: DefaultStrT, b: T2) -> None: ...
 
-   reveal_type(Foo(1, ""))       # type is Foo[int, str]
-   reveal_type(Foo[int](1, ""))  # type is Foo[int, str]  # example of partial specialisation
-   Foo[int]("", 1)               # Invalid: Foo[str, int] cannot be assigned to self: Foo[int, int] in Foo.__init__
+   reveal_type(Foo(1, ""))  # type is Foo[int, str]
+   Foo[int](1, "")          # Invalid: Foo[int, str] cannot be assigned to self: Foo[int, int] in Foo.__init__
+   Foo[int]("", 1)          # Invalid: Foo[str, int] cannot be assigned to self: Foo[int, int] in Foo.__init__
 
 When using a ``TypeVarLike`` as the default to another ``TypeVarLike``.
 Where ``T1`` is the default for ``T2`` the following rules apply.
 
+Note
+''''
+Currently ``TypeVarTuple``\s as defaults are not supported due to
+:ref:`scoping-rules` not allowing usage of ``TypeVarLikes`` from outer
+scopes and multiple ``TypeVarTuple``\s cannot appear in the same
```

Comment:
I think you mean to say that "TypeVarTuple's from outer scopes are not supported in default types". The scoping rules don't dictate this, but there are other good reasons to disallow this.

Also, I'm not sure what you mean by "multiple TypeVarTuples cannot appear in the same signature". Normally, "signature" refers to the parameter list of a function. Are you referring here to the type expression used for the default type? Why is it illegal for multiple TypeVarTuples to appear here? Is it legal for the same TypeVarTuple to appear multiple times, as in `tuple[int, *Ts] | tuple[str, *Ts]`?

Reply:
The scoping rules don't solely mean that they aren't allowed in defaults, it is also the combination that there aren't any cases where you can have a TypeVarTuple default to another and have be it resolved because you can't have
```py
class Foo[*Ts, *Us = *Ts]: ...
```
or because you can't have defaults in functions.

Unless I'm missing something there aren't any cases where you can use a TypeVarTuple with a TypeVarTuple default so they should just be disallowed.